debian/control

```diff
@@ -2,14 +2,16 @@ Source: rdiff-backup
 Section: utils
 Priority: optional
 Maintainer: Otto Kekäläinen <otto@debian.org>
-Build-Depends: debhelper (>= 11),
+Build-Depends: asciidoctor,
```

Comment:
Since 2.2.2 asciidoctor is an optional dependency on request of some other packagers, to avoid pulling too big of a dependency (ruby, etc), hence the man pages are pre-built and part of the sdist package (but they are not in the Git repo, on purpose, so not sure it applies to your case because I think you build from Git).

Reply:
Well this dependency is required for a full build, thus having it as a build dependency is correct.

This is not a run-time dependency.

Reply:
Fine wwith me, you already had my approval, I'll merge once the tests have gone through.